avocado_virt/plugins/virt_bootstrap.py

```diff
@@ -72,12 +72,12 @@ def run(self, args):
         if actual_sha1 != sha1:
             if actual_sha1 == '0':
                 LOG.debug('JeOS could not be found at %s. Downloading '
-                          'it (192 MB). Please wait...', jeos_dst_path)
+                          'it (204 MB). Please wait...', jeos_dst_path)
```

Comment:
As the image is something that changes very seldomly, not having to contact the internet to do get the size felt like a reasonable decision to make at the time.